### PR TITLE
[quant][core] Refactor qat conv implementation to use the same _ConvNd as base class

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #74506
* __->__ #74505

Summary:
att, this is to make supporting conv1d easier in future PR

Test Plan:
python test/test_quantization.py TestQuantizeFx.test_conv_bn_relu

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D35031523](https://our.internmc.facebook.com/intern/diff/D35031523)

### CODE REVIEW

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35031523).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35031523).

File: torch/nn/qat/modules/conv.py [hunk 1]
```diff
@@ -85,7 +84,57 @@ def to_float(self):
...
+    Attributes:
+        weight_fake_quant: fake quant module for weight
+    """
+    _FLOAT_MODULE = nn.Conv2d
+
+    def __init__(self,
+                 in_channels: int,
+                 out_channels: int,
+                 kernel_size: Tuple[int, ...],
+                 stride: Tuple[int, ...]=1,
```
Comment (vkuzo) [inline torch/nn/qat/modules/conv.py:108]: nit: lint everywhere

**Review by vkuzo — APPROVED** [review]

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35031523).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35031523).

**Pull request closed without merging**